src/Pages/LoginPage/LoginPage.js

```diff
@@ -1,7 +1,12 @@
 import 'font-awesome/css/font-awesome.min.css';
 import InputField from '../../Atoms/InputField/InputField';
+<<<<<<< HEAD
```

Comment:
it looks like lines with ====== or <<<<<<< or >>>>>> stayed after resolving conflicts

Reply:
Fixed. Stange...